aggregator/internal/pkg/server.go

```diff
@@ -43,35 +45,57 @@ func (agg *Aggregator) ServeOperators() error {
 //   - 0: Success
 //   - 1: Error
 func (agg *Aggregator) ProcessOperatorSignedTaskResponse(signedTaskResponse *types.SignedTaskResponse, reply *uint8) error {
-
-	agg.AggregatorConfig.BaseConfig.Logger.Info("New task response", "taskResponse", signedTaskResponse)
-
-	if _, ok := agg.OperatorTaskResponses[signedTaskResponse.BatchMerkleRoot]; !ok {
+	agg.AggregatorConfig.BaseConfig.Logger.Info("New task response",
```

Comment:
not really important for now but we should use directly `agg.Logger`. We can make an issue for solving this later in all the code